src/components/Input.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled'
+
+type InputProps = {
+  placeholder?: string
+  placeholderSize?: string
+  placeholderColor?: string
+  width?: string
+  height?: string
+  borderColor?: string
+  borderWidth?: string
+  inputTextColor?: string
+  inputTextSize?: string
+  inputBackgroundColor?: string
+  borderRadius?: string
+}
+
+const Input = ({
+  placeholder,
+  placeholderSize,
+  placeholderColor,
+  width,
+  height,
+  borderColor,
+  borderWidth,
+  borderRadius,
+  inputTextColor,
+  inputTextSize,
+  inputBackgroundColor,
+}: InputProps) => {
```

Comment:
```suggestion
}: Partial<InputProps>) => {
```
InputProps는 optional을 제거하고 Partial 유틸함수를 사용하는 게 낫지 않을까요?